public/app/features/variables/textbox/TextBoxVariableEditor.tsx

```diff
@@ -22,18 +20,5 @@ export function TextBoxVariableEditor({ onPropChange, variable: { query } }: Pro
   const onChange = useCallback((e: FormEvent<HTMLInputElement>) => updateVariable(e, false), [updateVariable]);
   const onBlur = useCallback((e: FormEvent<HTMLInputElement>) => updateVariable(e, true), [updateVariable]);
 
-  return (
-    <>
-      <VariableLegend>Text options</VariableLegend>
-      <VariableTextField
-        value={query}
-        name="Default value"
-        placeholder="default value, if any"
-        onChange={onChange}
-        onBlur={onBlur}
-        width={30}
-        testId={selectors.pages.Dashboard.Settings.Variables.Edit.TextBoxVariable.textBoxOptionsQueryInputV2}
-      />
-    </>
-  );
+  return <TextBoxVariableForm value={query} onChange={onChange} onBlur={onBlur} />;
```

Comment:
Same here, we probably can safe variable updates using onBlur only. WDYT?
```suggestion
  return <TextBoxVariableForm textValue={query} onTextValueBlur={onBlur} />;
```